### DISCUSSION TITLE
关于GPU下的推理速度，单图竟然在2700ms，有遇到过吗？

### DISCUSSION THREAD

**Discussion by XiaoDongGuoGuo** [PaddlePaddle/PaddleOCR#13817] [Q&A] [answered]
环境：
cuda11.6 
T4 10G
win10
PaddleOCR v2.6
图片使用官方库中的：
![254](https://github.com/user-attachments/assets/22372bbd-8362-40ce-bf53-10e63cc105aa)
实际测试下来效果：
![image](https://github.com/user-attachments/assets/138b1fd5-5968-4803-b8b1-7759dc21c015)

**Comment by XiaoDongGuoGuo** [comment]
补充信息：
预测库
![Uploading image.png…]()

**Reply by XiaoDongGuoGuo** [reply]
![image](https://github.com/user-attachments/assets/768e4359-7159-4f67-887d-f33c5e747d78)
det耗时比较久

**Reply by UserWangZz** [reply]
一般而言，如果文本数量多的情况，性能瓶颈在识别模型，这个看起来可能检测模型确实性能有些慢

**Reply by XiaoDongGuoGuo** [reply, marked as answer]
问题已找到原因，模型冷启动原因，后面持续推理CT就下来了
